src/leveldb/port/port_posix.h

```diff
@@ -71,7 +71,7 @@
 
 #if defined(OS_ANDROID) && __ANDROID_API__ < 9
 // fdatasync() was only introduced in API level 9 on Android. Use fsync()
-// when targetting older platforms.
```

Comment:
Please don't change the files inside `src/leveldb` - they come verbatim from upstream, merging this change would make it harder to keep the subtree up to date